lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -21,6 +21,6 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $rout
       user_id: CurrentUser.id).save()
 
   @showDescriptionPlaceholder = ->
-    CurrentUser.isAdminOf(@group) and !@group.description
+    AbilityService.canAdminister(@group) and !@group.description
```

Comment:
naming suggestion, can we call this canAdministerGroup .. I'm assuming it does not work for discussions etc..